packages/help-center/src/components/help-center-notice.tsx

```diff
@@ -1,10 +1,10 @@
-import { SupportActivity } from '@automattic/data-stores';
 import { localizeUrl, useLocale, getRelativeTimeString } from '@automattic/i18n-utils';
 import { ExternalLink } from '@wordpress/components';
 import { createInterpolateElement } from '@wordpress/element';
 import { __, sprintf } from '@wordpress/i18n';
 import { Icon, info } from '@wordpress/icons';
-import type { AnalysisReport } from '@automattic/data-stores';
+import { SupportActivity } from '../data/types';
```

Comment:
I think we can just move this to the next line, since it's also a type and imports from the same module.